bin/getInfoRippled.sh

```diff
@@ -0,0 +1,47 @@
+ #!/bin/bash
+
+export TMP_LOC="/tmp/ripple_info"
+if [ ! -d "${TMP_LOC}" ]; then
+       sudo mkdir "${TMP_LOC}"
+fi
+
+rm -f "${TMP_LOC}/*"
+cd ${TMP_LOC}
+echo "$ls"
+cd ~
+
+# Send output from this script to a log file
+## this captures any messages
+## or errors from the script itself
+
+export LOG_FILE="${TMP_LOC}/get_info.log"
+exec 3>&1 1>>"${LOG_FILE}" 2>&1
+
+## Send all stdout files to /tmp
+
+/opt/ripple/bin/rippled server_info > "${TMP_LOC}/server_info.txt"
+df -h                               > "${TMP_LOC}/free_disk_space.txt"
+cat /proc/meminfo                   > "${TMP_LOC}/amount_mem.txt"
+cat /proc/swaps                     > "${TMP_LOC}/swap_space.txt"
+ulimit -a                           > "${TMP_LOC}/reported_current_limits.txt"
+
+export TMP_CONF="${TMP_LOC}/rippled.cfg"
+
+sudo cp /etc/opt/ripple/rippled.cfg "${TMP_CONF}"
+
+export SEED_LINE_NUM=$(grep -n "\[validation_seed\]" "${TMP_CONF}" | tail -n 1 | awk -F ':' '{print $1}')
+export SEED_LINE_NUM=$((SEED_LINE_NUM + 1))
+
+awk -v SEED_LINE_NUM=${SEED_LINE_NUM} 'NR != SEED_LINE_NUM' "${TMP_CONF}" > ${TMP_LOC}/cleaned_rippled_cfg.txt
+
+if [ "$(cat /sys/block/xvda/queue/rotational)" = "0" ]
```

Comment:
Correct. This is specific to the device. There's no guarantee that a fixed block device will contain the information we need (e.g. what if the root is an SSD? We care about what Ripple is running on).

I'm discussing options for improving this with @johannakate.